api/src/main/java/org/pmiops/workbench/impersonation/ImpersonatedWorkspaceServiceImpl.java

```diff
@@ -26,7 +27,7 @@
  */
 @Service
 public class ImpersonatedWorkspaceServiceImpl implements ImpersonatedWorkspaceService {
-  private static final Logger LOGGER =
+  private static final Logger logger =
```

Comment:
https://stackoverflow.com/questions/1417190/should-a-static-final-logger-be-declared-in-upper-case